install_config/master_node_configuration.adoc

```diff
@@ -312,43 +312,43 @@ for the changes to take effect. See xref:master-node-config-restart-services[Res
 
 | Parameter Name | Description
 
-|`*AdmissionConfig*`
+|`*admissionConfig*`
 |Contains the xref:../architecture/additional_concepts/admission_controllers.adoc#architecture-additional-concepts-admission-controllers[admission control plug-in] configuration. {product-title} has a configurable list of admission controller plug-ins that are triggered whenever API objects are created or modified. This option allows you to override the default list of plug-ins; for example, disabling some plug-ins, adding others, changing the ordering, and specifying configuration. Both the list of plug-ins and their configuration can be controlled from Ansible.
 
-|`*APIServerArguments*`
+|`*apiServerArguments*`
 |Key-value pairs that will be passed directly to the Kube API server that match
 the API servers' command line arguments. These are not migrated, but if you
 reference a value that does not exist the server will not start. These values
 may override other settings in `*KubernetesMasterConfig*`, which may cause
-invalid configurations. Use `APIServerArguments` with the `event-ttl` value to store events in etcd. The default is `2h`, but it can be set to less to prevent memory growth:
+invalid configurations. Use `apiServerArguments` with the `event-ttl` value to store events in etcd. The default is `2h`, but it can be set to less to prevent memory growth:
 
 ----
 apiServerArguments:
   event-ttl:
   - "15m"
 ----
 
-|`*ControllerArguments*`
+|`*controllerArguments*`
 |Key-value pairs that will be passed directly to the Kube controller manager
 that match the controller manager's command line arguments. These are not
 migrated, but if you reference a value that does not exist the server will not
 start. These values may override other settings in `*KubernetesMasterConfig*`,
 which may cause invalid configurations.
 
-|`*DefaultAdmissionConfig*`
+|`*defaultAdmissionConfig*`
 |Used to enable or disable various admission plug-ins. When this type is present
 as the *configuration* object under `*pluginConfig*` and if the admission
 plug-in supports it, this will cause an *off by default* admission plug-in to be
 enabled.
 
-|`*PluginConfig*`
+|`*pluginConfig*`
 |Allows specifying a configuration file per admission control plug-in.
 
-|`*PluginOrderOverride*`
+|`*pluginOrderOverride*`
 |A list of admission control plug-in names that will be installed on the master.
```

Comment:
Bug comment said below for update:
```
admissionConfig and all its sub-params are placed in the same table column. At first glance, readers don't understand they are same level fields or parent-child relationship.
```
Checked 3.11 env, the level hierarchy is:
```
admissionConfig:
  pluginConfig:
    ...
  pluginOrderOverride: null
```
CC @wangke19